website/i18n/zh-cn/docusaurus-plugin-content-docs/version-2.3.json

```diff
@@ -0,0 +1,10 @@
+{
+  "version.label": {
+    "message": "2.3",
+    "description": "The label for version 2.3"
+  },
+  "sidebar.docs.category.APISIX™️": {
+    "message": "APISIX™️",
+    "description": "The label for category APISIX™️ in sidebar docs"
+  }
+}
```

Comment:
please update your code editor, it can help you add a blank line at the end of the file automatically.

Reply:
These files are generated by docusaurus cli, I didn't change it in my code editor. Maybe we can use husky* to format codes when commit.

* [husky GitHub repo](https://github.com/typicode/husky)